src/Sylius/Bundle/CoreBundle/Resources/config/app/workflow/sylius_order_payment.yaml

```diff
@@ -0,0 +1,64 @@
+framework:
+    workflows:
+        !php/const Sylius\Component\Core\OrderPaymentTransitions::GRAPH:
+            type: state_machine
+            marking_store:
+                type: method
+                property: paymentState
+            supports:
+                - Sylius\Component\Core\Model\OrderInterface
+            initial_marking: !php/const Sylius\Component\Core\OrderPaymentStates::STATE_CART
+            places:
+                - !php/const Sylius\Component\Core\OrderPaymentStates::STATE_CART
+                - !php/const Sylius\Component\Core\OrderPaymentStates::STATE_AWAITING_PAYMENT
+                - !php/const Sylius\Component\Core\OrderPaymentStates::STATE_PARTIALLY_AUTHORIZED
+                - !php/const Sylius\Component\Core\OrderPaymentStates::STATE_AUTHORIZED
+                - !php/const Sylius\Component\Core\OrderPaymentStates::STATE_PARTIALLY_PAID
+                - !php/const Sylius\Component\Core\OrderPaymentStates::STATE_CANCELLED
```

Comment:
What do you think moving `cancelled` after `authorized`, but before`partially_paid`? Same for transition.

It never made sense why `cancelled` is between two `partially_paid` and `paid` states, though it's correct to be after authorization states as they allow cancel.

Reply:
![image](https://github.com/Sylius/Sylius/assets/33687392/733fadc0-8f9d-48e3-a5fe-f1bdd89cc5a4)
Yes, I think it is a good point